bridgev2/database/upgrades/21-disappearing-message-fkey.postgres.sql

```diff
@@ -0,0 +1,7 @@
+-- v21: Add foreign key constraint from disappearing_message.mx_room to portals.mxid
```

Comment:
oh the compat flags are missing too

```suggestion
-- v21 (compatible with v9+): Add foreign key constraint from disappearing_message.mx_room to portals.mxid
```